CMakeLists.txt

```diff
@@ -13,6 +13,7 @@ add_executable(${PROJECT_NAME}
 target_include_directories(${PROJECT_NAME} PUBLIC
     ${PROJECT_SOURCE_DIR}/src
     ${PROJECT_SOURCE_DIR}/espert-core/src
+    ${PROJECT_SOURCE_DIR}/espert-core/src/Core
```

Comment:
Dodanie katalogu jako include directory dodaje również wszystkie jego podkatalogi, więc jak mamy `${PROJECT_SOURCE_DIR}/espert-core/src` to `${PROJECT_SOURCE_DIR}/espert-core/src/Core` jest zbędne. Jedyną różnicę jaką to robi jest to, że bez tej linijki musisz podawać w includach katalog `Core`, np `#include "Core/ClientPoint.hh"` zamiast `#include "ClientPoint.hh"`.

Reply:
No właśnie po to dałem tą linijkę żeby pominąć "core" ale mogę to usunąć jak wolisz

Reply:
Jeśli świadomie chciałeś tak to dodać to git.